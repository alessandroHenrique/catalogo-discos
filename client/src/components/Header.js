import React from 'react';

const Header = (props) => (
  <div className="header">
    <div className="container">
      <h1 className="header__title">{props.title}</h1>
    </div>
  </div>
);

Header.defaultProps = {
  title: 'Collection App'
};

export default Header;
