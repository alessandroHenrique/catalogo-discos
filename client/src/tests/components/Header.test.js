import Header from '../../components/Header';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Header', () => {
  it('should render Header', () => {

    const component = renderer.create(<Header />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
