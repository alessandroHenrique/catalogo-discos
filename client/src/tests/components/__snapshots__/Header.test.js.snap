// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render Header 1`] = `
<div
  className="header"
>
  <div
    className="container"
  >
    <h1
      className="header__title"
    >
      Collection App
    </h1>
  </div>
</div>
`;
